app/views/shared/provider/_header.html.slim

```diff
@@ -1,53 +1,42 @@
-          - else
-            ' Signed in to
-          => current_user.account.name
-          ' as
-          = current_user.display_name
-          | .
-
-        = fixed_width_icon_link_to 'Sign Out', 'times', provider_logout_path, class: 'PopNavigation-link', title: "Sign Out #{current_user.username}"
-
-
-  .PopNavigation.PopNavigation--docs
-    a.PopNavigation-trigger.u-toggler class=('is-current' if active_menu?(:topmenu, :help)) href="#documentation-menu" title="Documentation"
-      => icon('question-circle')
-    ul#documentation-menu.PopNavigation-list.u-toggleable
-      li.PopNavigation-listItem
-        = fixed_width_icon_link_to 'Customer Portal', 'external-link', '//access.redhat.com/products/red-hat-3scale', class: 'PopNavigation-link', target: '_blank'
-      li.PopNavigation-listItem
-        = fixed_width_icon_link_to '3scale API Docs', 'puzzle-piece', provider_admin_api_docs_path, class: 'PopNavigation-link'
-      li.PopNavigation-listItem
-        = fixed_width_icon_link_to 'Liquid Reference', 'code', provider_admin_liquid_docs_path, class: 'PopNavigation-link'
-      - if saas?
-        li.PopNavigation-listItem
-          = fixed_width_icon_link_to "What's new?", 'leaf', '//access.redhat.com/articles/3107441#newfeaturesenhancements', class: 'PopNavigation-link', target: '_blank'
-
-  .PopNavigation.PopNavigation--account
-    - link = can?(:manage, current_account) ? provider_admin_account_path : edit_provider_admin_user_personal_details_path
-
-    a.PopNavigation-trigger class=('is-current' if active_menu?(:topmenu, :account)) href="#{link}"  title="Account Settings"
-      => icon('cog')
-
-  // HACK of the month. See features/support/current_user.rb if you are hungry for reasons
-  div class="username" style="display:none;"
-    = current_user.username
+- class_header = current_account.master ? 'Header Header--master' : 'Header'
+- header_with_logo = active_menu?(:main_menu, :dashboard) ? 'Header-logo--withIcon' : ''
+
+- account_settings_link = can?(:manage, current_account) ? provider_admin_account_path : edit_provider_admin_user_personal_details_path
+- account_settings_class = active_menu?(:topmenu, :account) ? 'is-current' : ''
+
+- docs_link = active_menu?(:topmenu, :help) ? 'is-current' : ''
+- isSaas = saas? ? 'true' : 'false'
+- docs_links_class = 'PopNavigation-link'
+- customer_portal_link = '//access.redhat.com/products/red-hat-3scale'
+- api_docs_link = provider_admin_api_docs_path
+- liquid_reference_link = provider_admin_liquid_docs_path
+- what_is_new_link = '//access.redhat.com/articles/3107441#newfeaturesenhancements'
+
+- avatar_link = active_menu?(:topmenu, :help) ? 'is-current' : ''
+- impersonated = impersonating? ? 'true' : nil
+- account_name = current_user.account.name
+- display_name = current_user.display_name
+
+- context_switcher_data={'currentApi' => @service, 'apis' => api_selector_services, 'activeMenu' => active_menu, 'audienceLink' => audience_link}.to_json
```

Comment:
Why not preserving the same name to make the fewest changes?
https://github.com/3scale/porta/pull/618/files#diff-6b72418e6c4006fa761c6fd926301800R6

Reply:
Because is not a node anymore, now is pure data